CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## HEAD
 
+* Added contexts feature. Thanks to @apurvis.
```

Comment:
Oh what I usually do is put new feature/bug fixes in HEAD, and then when I cut a new version, I'll move the new stuff into that version. Since sometimes I collect a few features or bug fixes before doing a version.